client/components/EventActionConfirmation/UpdateMethodSelection.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './style.scss'
+import { EventUpdateMethodField } from '../fields/index'
+import { Field } from 'redux-form'
+import { RelatedEvents, RelatedPlannings } from '../index'
+
+export const UpdateMethodSelection = ({
+        showMethodSelection,
+        updateMethodLabel,
+        relatedPlannings,
+        relatedEvents,
+        dateFormat,
+        action,
+        handleSubmit,
+    }) => {
+    return (
+        <div>
+            { showMethodSelection &&
+            <form onSubmit={handleSubmit}>
+                <div className="MethodSelect">
+                    <span>
+                        <strong>This event is a recurring event!</strong>
+                    </span>
+
+                    <Field name="update_method"
+                           component={EventUpdateMethodField}
+                           label={updateMethodLabel}/>
+                </div>
+
+                <button type="submit" style={{ visibility: 'hidden' }}>Submit</button>
+            </form>}
+
+            // We are showing related planning irrespective of it being a recurring event or not
+            { relatedPlannings.length > 0 && (
+                <div>
+                    <div className="sd-alert sd-alert--hollow sd-alert--alert">
+                        <strong>This will also {action} the following planning items</strong>
+                        <RelatedPlannings
+                            plannings={relatedPlannings}
+                            openPlanningItem={true}
+                            short={true} />
+                    </div>
+                </div>
+            )}
+
+            { showMethodSelection && relatedEvents.length > 0 && (
+                <div>
+                    <div className="sd-alert sd-alert--hollow sd-alert--alert">
+                        <strong>This will also update the following events</strong>
```

Comment:
Change
`this will also update the following events`
to
`this will also {action} the following events`